src/bridged_logging.rs

```diff
@@ -186,6 +198,37 @@ impl rs_log::Log for GlibLogger {
     fn flush(&self) {}
 }
 
+/// Registers a glib log handler which routes all logging messages to the
+/// [`log crate`](https://crates.io/crates/log).
+///
+/// In order to use this type, `glib` must be built with the `log` feature
+/// enabled.
+///
+/// Use this function if you want to use the log crate as the main logging
+/// output in your application, and want to route all logging happening in
+/// glib to the log crate. If you want the opposite, use [`GlibLogger`](struct.GlibLogger.html).
+///
+/// NOTE: This should never be used when [`GlibLogger`](struct.GlibLogger.html) is
+/// registered as a logger, otherwise a stack overflow will occur.
+///
+/// ```no_run
+/// register_glib_to_logcrate_logger();
+/// ```
+pub fn register_glib_to_logcrate_logger() {
```

Comment:
The name is a bit weird with the `crate` in there. Not sure what would be a better name :)

Or maybe we could only expose the actual logging function and let users call `log_set_default_handler()` with it?

@GuillaumeGomez What do you think?

Reply:
I'm also definitely not happy with the name (long and clumsy).
Sadly there is an overload on the "log" / "logger" terms which is not easy to untangle, and the crate approach means that there is no official Rust logging facility, while at the same time the logging crate is called just... "log", adding to the overload and confusion.
Suggestions are definitely welcome.

(s far as the rest goes, previous comments have been addressed and CI is green)

Reply:
Throwing some options:
- `register_rust_log_handler` / `set_rust_log_handler` / `set_default_rust_log_handler`
- `rust_log_handler` (and the application code calls `log_set_default_handler(rust_log_handler)`
- `redirect_rust_logs_to_glib`
- `setup_glib_logging` (might create confusion with `GlibLogger`)

Reply:
I think I prefer the second option. @GuillaumeGomez ?

Reply:
Fine by me!

Reply:
Let's go with that then or what do you think @xanathar ? :)

Reply:
Sure! I'll push it asap